### ISSUE TITLE
A bug about loss function

### ISSUE THREAD

**Issue by Zhangsihao-Yang** [EdwardSmith1884/GEOMetrics#12]
Check the line below:
https://github.com/EdwardSmith1884/GEOMetrics/blob/09471d70f441482319543bb5214eaaab38f64507/GEOMetrics.py#L139
07/GEOMetrics.py#L139
Should it be: 
`total_loss += surface_loss`

**Comment by EdwardSmith1884** [comment]
It can be either I guess depending on what you want to log
